libs/csm/rest/csm_rest_quota.py

```diff
@@ -62,15 +66,20 @@ def get_user_quota(self, uid, quota_type, **kwargs):
         self.log.info("Get user quota request successfully sent...")
         return response
 
-    def iam_user_quota_payload(self):
+    def iam_user_quota_payload(self): #pylint disable=no-self-use
         """
         Create IAM user quota payload
         """
         payload = {}
+        quota_type = "user"
+        enabled = True
+        max_size = "5G"
+        max_objects = 1000
```

Comment:
Please accept all these as variable and they should come from test config

Reply:
accepted